staging/src/k8s.io/client-go/dynamic/client.go

```diff
@@ -280,34 +275,6 @@ func (rc *ResourceClient) UpdateStatus(obj *unstructured.Unstructured) (*unstruc
 	return result, err
 }
 
-// GetScale takes name of the resource, and returns the corresponding autoscalingv1.Scale object, and an error if there is any.
-func (rc *ResourceClient) GetScale(name string, opts metav1.GetOptions) (*autoscalingv1.Scale, error) {
-	result := new(autoscalingv1.Scale)
-	err := rc.cl.Get().
-		NamespaceIfScoped(rc.ns, rc.resource.Namespaced).
-		Resource(rc.resource.Name).
-		Name(name).
-		SubResource("scale").
-		VersionedParams(&opts, scheme.ParameterCodec).
-		Do().
-		Into(result)
-	return result, err
-}
-
-// UpdateScale takes the top resource name and the representation of a scale and updates it. Returns the server's representation of the scale, and an error, if there is any.
-func (rc *ResourceClient) UpdateScale(name string, scale *autoscalingv1.Scale) (*autoscalingv1.Scale, error) {
```

Comment:
no need for update?

Reply:
`UpdateStatus` is still present.

Did you mean `UpdateScale`? It's part of the scale client - https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/scale/client.go#L154.

Reply:
You are right. Was commenting from the mobile, misread the func name.